### DISCUSSION TITLE
how to center popover to the middle of the element

### DISCUSSION THREAD

**Discussion by DiegoCamposYanez** [elrumordelaluz/reactour#479] [Q&A]
Hello, I am using V2, I would like to center the popover with respect to the element, something like this:

![image](https://user-images.githubusercontent.com/72150630/173253572-af4a0ac6-e741-4541-bf4c-ef0329079121.png)

I currently have this setting in the global styles: 

`styles={{
        popover: base => ({
          ...base,
          borderRadius: 20,
          // center popover with respect to the element
        }),
      }}`

For this operation I need to modify the global styles? Any idea or suggestion for this?

Thank you very much for everything, greetings and sorry for my English.

**Comment by elrumordelaluz** [comment]
Hi @DiegoCamposYanez, thank you for your interesting question.

You could solve this situations taking advantage of the _Popover_ [position](https://github.com/elrumordelaluz/reactour/tree/master/packages/popover#position-position) prop, which accepts a function with all the information necessary to make custom calculations. You could pass the [position prop](https://github.com/elrumordelaluz/reactour/tree/master/packages/tour#position-position) globally in the _TourProvider_, or in case you needed, customizing it in a [specific step](https://codesandbox.io/s/reactour-tour-demo-using-react-bootstrap-modal-forked-35b4dh?file=/steps.js).

[Here is a working example](https://codesandbox.io/s/reactour-tour-demo-using-react-bootstrap-modal-forked-35b4dh?file=/steps.js) with a globally custom `position`, overridden for step 2.

Let me know in case some part isn't clear.
